<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modConge</class>
 <widget class="QDialog" name="modConge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conge/Modifier</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Agent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type conge</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Motif</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date debut</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Date fin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Date retour effectif</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="motifEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="typecongeCombo"/>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="datedebutEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="datefinEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="dateretoureffectifEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="agentEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btAgent">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btValider_2">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btValider_2</sender>
   <signal>clicked()</signal>
   <receiver>modConge</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
